src/containers/Results/index.js

```diff
@@ -1,38 +1,275 @@
+      0
+    );
+
+    const samplesAdded = results.reduce(
+      (sum, result) =>
+        dataSet[result.accession_code]
+          ? sum + dataSet[result.accession_code].length
+          : sum,
+      0
+    );
+
+    return (
+      <div className="results">
+        <Helmet>
+          <title>refine.bio - Results</title>
+        </Helmet>
+
+        <BackToTop />
+        <div className="results__search">
+          <SearchInput onSubmit={this.handleSubmit} searchTerm={searchTerm} />
+        </div>
+
+        {isLoading || this.state.isLoading ? (
+          <div className="loader" />
+        ) : !results.length ? (
+          <EmptyStates searchTerm={searchTerm} />
+        ) : (
+          <div className="results__container">
+            <div className="results__filters">
+              <ResultFilters
+                toggledFilter={toggledFilter}
+                filters={filtersData}
+                appliedFilters={this.state.filters}
+              />
+            </div>
+            <div className="results__list">
+              <div className="results__top-bar">
+                {results.length ? <NumberOfResults /> : null}
+                {totalSamplesOnPage - samplesAdded === 0 ? (
+                  <RemoveFromDatasetButton
+                    totalAdded={totalSamplesOnPage}
+                    handleRemove={this.handlePageRemove}
+                  />
+                ) : (
+                  <AddToDatasetButton
+                    addMessage="Add Page to Dataset"
+                    handleAdd={() => {
+                      addExperiment(results);
+                    }}
+                    samplesInDataset={samplesAdded}
+                    buttonStyle="secondary"
+                  />
+                )}
+              </div>
+              {results.map((result, i) => (
+                <Result
+                  key={i}
+                  result={result}
+                  addExperiment={addExperiment}
+                  removeExperiment={removeExperiment}
+                  dataSet={dataSet}
+                />
+              ))}
+              <Pagination
+                onPaginate={this.handlePagination}
+                totalPages={totalPages}
+                currentPage={currentPage}
+              />
+            </div>
+          </div>
+        )}
+      </div>
+    );
+  }
+}
+Results = connect(
+  ({
     search: {
       results,
-      organisms,
       filters,
-      appliedFilters,
       pagination,
       searchTerm,
       isSearching
     },
     download: { dataSet }
-  } = state;
-  return {
+  }) => ({
     results,
-    organisms,
     filters,
-    appliedFilters,
     pagination,
     searchTerm,
     dataSet,
     isLoading: isSearching,
-    searchInputForm: state.form.searchInput
-  };
-};
-
-const ResultsContainer = connect(mapStateToProps, {
+  }), {
   ...resultsActions,
   ...downloadActions
 })(Results);
 
-export default ResultsContainer;
+export default Results;
+
```

Comment:
I think one or two blank lines would be good enough. Any reason why there are 5 here?

Reply:
I agree, it seems I hadn't configured correctly vscode to apply prettify to the code. I pushed changes now to fix this. Thanks!